src/components/badge/experimental_badge/index.ts

```diff
@@ -28,4 +28,10 @@
  * under the License.
  */
 
-export { OuiBetaBadge, OuiBetaBadgeProps } from './beta_badge';
+// TODO: remove beta badge in next major release
```

Comment:
it's preferable to create a release-tagged issue and link to it in the comment (and actually identify the particular release)